packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -111,6 +142,24 @@ export class PluginLedgerConnectorCorda
 
       endpoints.push(endpoint);
     }
+
+    {
+      const opts: IRunTransactionEndpointV1Options = {
```

Comment:
Should incorporating this be the part of the future issue? Because I went through the LOCAL and REMOTE import type logic and I didnt understand it in the first go.

Reply:
@jagpreetsinghsasan That works too, please create a follow-up issue and make sure to reference this conversation for context/info.